packages/google_maps_flutter/google_maps_flutter/pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: google_maps_flutter
 description: A Flutter plugin for integrating Google Maps in iOS and Android applications.
 homepage: https://github.com/flutter/plugins/tree/master/packages/google_maps_flutter/google_maps_flutter
-version: 2.0.1
+version: 2.0.2
```

Comment:
Sorry, I hadn't looked at this closely enough; you've updated the version and changelog of the app-facing package, rather than the platform interface package where you made the change. You need to update the version of the package that your changes are in.